ansible_runner/runner_config.py

```diff
@@ -369,6 +386,15 @@ def build_process_isolation_temp_dir(self):
         os.chmod(path, stat.S_IRUSR | stat.S_IWUSR | stat.S_IXUSR)
         return path
 
+    def wrap_args_with_cgexec(self, args):
+        '''
+        Wrap existing command line with cgexec in order to profile resource usage
+        '''
+        # TODO: Should only wrap if `self.execution_mode == ExecutionMode.ANSIBLE_PLAYBOOK` (see example below)
+        new_args = ['cgexec', '--sticky', '-g', 'cpuacct,memory,pids:{}/{}'.format(self.resource_profiling_base_cgroup, self.ident)]  # Is sticky needed?
```

Comment:
I dunno... is sticky needed? 😄

Reply:
Yep! This will ensure that any subprocesses that get spawned will run under the same cgroup as the parent.

https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/6/html/resource_management_guide/starting_a_process

> You can also add the --sticky option before the command to keep any child processes in the same cgroup.

Reply:
Then I think we can take that comment off.

Reply:
Should be removed now